src/actions/kloudio/kloudio.ts

```diff
@@ -0,0 +1,178 @@
+import * as https from "request-promise-native"
+
+import * as uuid from "uuid"
+import * as winston from "winston"
+import * as Hub from "../../hub"
+
+const signedUrl = "https://api.kloud.io/v1/tools/signed-url-put-object?key="
+const API_URL = "https://ecuoyevlzf.execute-api.us-west-2.amazonaws.com/prod/dest/send"
```

Comment:
Is this fine to expose on an open source project? If this is fine with you I'll merge it but maybe a comment with who Looker should email if someone makes a PR to change this would be a good safety measure

Reply:
That or you can give us a current url to deploy in an environment variable

Reply:
In fact I would prefer if you made this an environment variable that we could load on actionhub startup.

Reply:
Environment variables work for us if they can only be accessed by only our code & Looker's code. If it can be accessed by other people's code then the current setup works fine for us security wise. You can send an email to developer@kloud.io when someone raises a PR to change them. This is a monitored email.

Reply:
Let's make this an environment variable that we control, like `KLOUDIO_API_URL`. We will keep this in a secure store that is only read at startup of that server. If it needs to be update KloudIO can contact Looker to get it updated

Reply:
It will not be accessed by any other actions

Reply:
Ok I'll make the change and update the PR. How should the the values of these variables sent? with this PR or through an email?

Reply:
Hi! Just to clarify, both the variables(signedUrl, API_URL) need to be replaced by environment variables right? or just the API_URL?